MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/maps/renderer/glsurfaceview/MapboxGLSurfaceView.java

```diff
@@ -432,6 +432,10 @@ boolean createSurface() {
         Log.e(TAG, "mEglConfig not initialized");
         return false;
       }
+      if (mEglContext == null) {
```

Comment:
Wonder what the outcome would be, if we skip init here black map? 
Also, to help debugging, I'd probably enriched logs a bit, like `Could not create surface for drawing, mEglContext not initialized`